core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html

```diff
@@ -4,75 +4,77 @@
                          activity-title="getExplorationTitle()"
                          activity-active="explorationIsCurrentlyHoveredOver">
 </learner-dashboard-icons>
-<md-card ng-class="[isCollectionPreviewTile ? 'oppia-activity-summary-tile-mobile' : 'oppia-activity-summary-tile', {'small-width': !isWindowLarge}, {'oppia-activity-playlist-tile': isPlaylistTile()}]"
-         style="position: relative; z-index: 5;">
-  <a ng-href="<[getExplorationLink()]>" target="<[openInNewWindow ? '_blank' : '_self']>">
-    <div class="title-section" style="background-color: <[getThumbnailBgColor()]>;">
-      <img class="thumbnail-image" ng-src="<[getCompleteThumbnailIconUrl()]>" alt="<[getCategory()]>" ng-style="isPlaylistTile() && {'top': '4px', 'width': '50px'}">
-      <h3 class="activity-title protractor-test-exp-summary-tile-title" ng-style="isPlaylistTile() && {'width': '93%'}">
-        <span ng-if="isWindowLarge"><[getExplorationTitle()|truncate:40]></span>
-        <span ng-if="!isWindowLarge"><[getExplorationTitle()|truncate:40]></span>
-      </h3>
+<div ng-class="{'oppia-activity-summary-tile-mobile-background-mask': isCollectionPreviewTile}" ng-click="closeOnClickingOutside($event)">
```

Comment:
I mentioned this in the previous review -- I suggest moving this container div to collection_player.html instead. It's not a part of the exploration summary tile directive and certainly shouldn't exist in other pages that the exploration summary tile is used in.

Reply:
Done